frontend/src/features/admin-form/preview/PreviewFormPage.stories.tsx

```diff
@@ -69,6 +69,21 @@ export default {
 const Template: Story = () => <PreviewFormPage />
 export const Default = Template.bind({})
 
+export const WithInstructions = Template.bind({})
+WithInstructions.parameters = {
+  msw: [
+    ...envHandlers,
+    getPreviewFormResponse({
+      delay: 0,
+      overrides: {
+        form: {
+          startPage: { paragraph: 'Fill in this mock form in this story.' },
```

Comment:
should we add an example with super long paragraph text to snapshot? Super super super long one. Like multiline~

Reply:
will add!